### DISCUSSION TITLE
Unable to delete resource: namespaces "default" is forbidden: this namespace may not be deleted

### DISCUSSION THREAD

**Discussion by juliohm1978** [argoproj/argo-cd#9634] [Q&A] [answered]
While experimenting with argocd, I included the following resource in the git repo:

```yaml
apiVersion: v1
kind: Namespace
metadata:
  name: default
```

In retrospect, I just wanted to keep the namespace annotations and labels synched with my git repo.

Now, I am unable to delete this resource from argocd because Kubernetes refuses the delete the default namespace.

![image](https://user-images.githubusercontent.com/4145456/173192971-c1d79d59-d609-4b89-a6cc-8e21ab6828ec.png)

The yaml resource has been removed from the git repo, but I am unable to sync argocd.

![image](https://user-images.githubusercontent.com/4145456/173193029-db5132eb-6ba9-4304-a113-99c166e1c4df.png)

Is there a way to remove the resource reference from argocd without having to actually delete the resource from the cluster?

**Comment by jannfis** [comment, marked as answer]
I guess to "unmanage" that resource, you would need to patch out the tracking label that Argo CD has applied to the live resource. Given a standard Argo CD configuration and that the resource was deleted in Git, you can run:

```
kubectl patch ns default --type=json -p='[{"op":"remove", "path":"/metadata/labels/app.kubernetes.io~1instance"}]
```

**Comment by juliohm1978** [comment]
Thank you! That does the trick and causes argocd to ignore the resource.

Best regards!

**Comment by ymktmk** [comment]
I did it in the following way
```
kubectl label ns default argocd.argoproj.io/instance-
```
